app/assets/javascripts/admin/addon/controllers/modals/admin-watched-word-test.js

```diff
@@ -16,16 +16,18 @@ export default Controller.extend(ModalFunctionality, {
     "isTag",
     "isLink"
   )
-  matches(value, regexpString, words, isReplace, isTag, isLink) {
-    if (!value || !regexpString) {
+  matches(value, regexpList, words, isReplace, isTag, isLink) {
+    if (!value || !regexpList.length) {
```

Comment:
We try to optimise for reading code rather than using the shortest code. Using `!array.length` is not as clear as `!array.length === 0`

```suggestion
    if (!value || regexpList.length === 0) {
```